### DISCUSSION TITLE
All data dissapearen on the site

### DISCUSSION THREAD

**Discussion by ukraine-sdg** [open-sdg/open-sdg#1487] [Q&A] [answered]
Our data dissapeared fro the site, but they are available in the data repository. Could you, please help to fix it?

**Comment by brockfanning** [comment, marked as answer]
@ukraine-sdg It looks like it is due to a mismatch of version numbers which is causing a Javascript error. First a bit of background: The functionality of Open SDG is split between two separate projects:

* https://github.com/open-sdg/open-sdg - This is the main project. Technically this can be considered a "Jekyll theme".
* https://github.com/open-sdg/jekyll-open-sdg-plugins/ - This is a "Ruby gem", and contains all of the complex logic for Open SDG.

The reason that these are separated is purely technical. It would be better if everything could be in a single project, but that's just the way that Jekyll themes work.

Since these projects are separate, you have to set their versions separately. You set the version of open-sdg in [your _config.yml file](https://github.com/ukraine-sdg/site/blob/develop/_config.yml#L4). And you set the version of jekyll-open-sdg-plugins in [your Gemfile](https://github.com/ukraine-sdg/site/blob/develop/Gemfile#L7).

As you can see, right now you have open-sdg set to "1.7.0-dev" and jekyll-open-sdg-plugins set to "1.5.0". This is a bit of a mismatch, and is probably the source of the problem.

In general we recommend that you use the latest release. In this case it's 1.6.0. However it so happens that jekyll-open-sdg-plugins received a hotfix release, so it is at 1.6.1. So I would recommend that you change those two files linked above, like so:

* [your _config.yml file](https://github.com/ukraine-sdg/site/blob/develop/_config.yml#L4): change the line to `remote_theme: open-sdg/open-sdg@1.6.0`
* [your Gemfile](https://github.com/ukraine-sdg/site/blob/develop/Gemfile#L7): change the line to `gem "jekyll-open-sdg-plugins", "1.6.1"`
